index.html

```diff
@@ -57,64 +57,35 @@ <h1>
-      <a href="https://w3c.github.io/remote-playback/">Remote Playback API</a>.
-      In addition, this Community Group incubates new additions to the APIs
-      defined in the Second Screen WG.
-    </p>
-    <p>
-      This work has three primary purposes:
-    </p>
-    <ul>
-      <li>
-        <em>Enable interoperability among implementations of the Presentation
-        API and Remote Playback API.</em> All browsers and displays that
-        implement the specifications should be able to support the APIs, with
-        strong security and authentication for the data exchanged among them.
-        Interoperability among controllers and receivers will increase the
-        number of opportunities for the Presentation API and Remote Playback
-        API to be useful. Interoperability will also create more uniform and
-        predictable API behavior across user agents, encouraging adoption of
-        the Presentation API and Remote Playback API by Web developers.
-      </li>
-      <li>
-        <em>Encourage implementation of the Presentation API and Remote
-        Playback API by browser vendors.</em> The specifications should reduce
-        the design work required to implement the APIs in a user agent.
-      </li>
-      <li>
-        <em>Establish complementary specifications for the Presentation API and
-        Remote Playback API.</em> The Presentation API and Remote Playback API,
-        by nature of their scope, leave many network details up to the
-        implementation. This includes how displays are discovered, how
-        presentations and remote playbacks are started, connected, and stopped,
-        and how the controller and receiver communicate. The specifications
-        will provide details on how the implementation requirements of the APIs
-        can be met and will clarify implementation issues that are not
-        addressed in the API specifications themselves.
-      </li>
-    </ul>
-    <p>
-      The scope of work proposed here goes beyond the current <a href=
+      The mission of the <a href=
+      "https://www.w3.org/community/webscreens/">Second Screen Community Group
+      (CG)</a> is to explore, incubate, and define interfaces that enable new
+      multi-display and multi-window computing user experiences on the Web.
+      This Community Group defines new additions to the <a href=
+      "https://w3c.github.io/presentation-api/">Presentation API</a> and
+      <a href="https://w3c.github.io/remote-playback/">Remote Playback API</a>,
+      and incubates the <a href=
+      "https://github.com/spark008/screen-enumeration/blob/master/EXPLAINER.md">
+      Screen Enumeration</a> and <a href=
+      "https://github.com/spark008/window-placement/blob/master/EXPLAINER.md">Window
+      Placement</a> APIs.
+    </p>
+    <p>
+      The scope of work for this Community Group extends beyond the current
+      <a href=
       "http://www.w3.org/2014/secondscreen/charter.html#scope">scope</a> of the
```

Comment:
Note to self: reference the latest WG charter here.

Reply:
Actually, since the WG charter 2020 is not yet deployed to its permanent location, I just dropped this link in https://github.com/webscreens/cg-charter/pull/20/commits/998b6383d9298f9e996a8f596908c2a0c6725013 for clarity.